R/pkg/R/functions.R

```diff
@@ -3548,7 +3548,7 @@ setMethod("row_number",
 
 #' array_contains
 #'
-#' Returns true if the array contain the value.
+#' Returns null if the array is null, true if the array contains the value, and false otherwise.
```

Comment:
Yea, I agree with being careful. For this PR, I followed the others. I skimmed again and it seems we have not used the notation for `None`, `True` and `False` in `functions.py`, and `NA, `TRUE` and `FALSE` in both `functions.R`.

I can grep and replace.